spec.bs

```diff
@@ -165,3 +168,17 @@ The {{HTMLFencedFrameElement/src}} IDL attribute must [=reflect=] the respective
 <h3 id=dimension-attributes>Dimension attributes</h3>
 
 This section details monkeypatches to [[!HTML]]'s <a href="https://html.spec.whatwg.org/multipage/embedded-content-other.html#dimension-attributes">Dimension attributes</a> section. This section will be updated to include <{fencedframe}> in the list of elements that the <code>[=width=]</code> and <code>[=height=]</code> dimension attributes apply to.
+
+<h3 id=coep>Patching [=Cross-Origin Resource Policy Internal Check=]</h3>
+
+In [[!Fetch]]'s [=cross-origin resource policy internal check=] steps, in the step
+
+> 5. Switch on policy:
+
+Where it currently states
+
+> If origin is <a>same origin</a> with response's <a for=response>URL</a>'s <a for=url>origin</a>, then return <b>allowed</b>.
+
+Modify it to say
+
+> If origin is <a>same origin</a> with response's <a for=response>URL</a>'s <a for=url>origin</a>, and [=request/destination=] is not a fenced frame, then return allowed.
```

Comment:
> is not a fenced frame

Can you wrap "fenced frame" in quotes, or whatever is appropriate for destinations? I think we use quotes and <code> or something, like https://fetch.spec.whatwg.org/#request-destination-script-like

Reply:
Looks like the convention is "<code>destination</code>", so I'll update this to match.